components/dropdown/demo/api.min.js

```diff
@@ -56,7 +56,7 @@ const t$1=globalThis,e$3=t$1.ShadowRoot&&(void 0===t$1.ShadyCSS||t$1.ShadyCSS.na
 
 /* eslint-disable line-comment-position, no-inline-comments, no-confusing-arrow, no-nested-ternary, implicit-arrow-linebreak */
 
-class AuroLibraryRuntimeUtils {
+let AuroLibraryRuntimeUtils$1 = class AuroLibraryRuntimeUtils {
```

Comment:
Was this auto-generated by a process?

Have we used the `$1` convention elsewhere?

Reply:
Yes this is a generated file. Planning to get these removed from git once I have my new build system in place.